docs/guides/tutorials/nextjs_pages_router.rst

```diff
@@ -1,10 +1,10 @@
-.. _ref_guide_nextjs:
+.. _ref_guide_nextjs_pages_router:
```

Comment:
I also like to update the label names, but it does require us to change any existing references to the old label. In this case, that would require a companion PR [in the web site repo](https://github.com/edgedb/edgedb.com/blob/master/content/blog/2023-08-04-keep-it-ryl-in-your-documentation/post.rst?plain=1#L94).

Reply:
https://github.com/edgedb/edgedb.com/pull/797

Reply:
This was the only reference I found. Let me know it I missed something